src/java_bytecode/jar_file.cpp

```diff
@@ -9,9 +9,9 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <cstring>
 #include <cassert>
 #include <unordered_set>
-
```

Comment:
re-added this line and one in front of documentation block for the function afterwards
no particular reason for having removed it